web/src/integration_url_modal.ts

```diff
@@ -55,6 +55,7 @@ export function show_generate_integration_url_modal(api_key: string): void {
             const checked = $(this).prop("checked");
             $topic_input.parent().toggleClass("hide", !checked);
         });
+        $("#integration-url-stream_widget").prop("disabled", true);
```

Comment:
Maybe have this line up where the dialog submit button is also disabled for reability.